api/pkg/apis/v1alpha1/vendors/catalogs-vendor.go

```diff
@@ -60,12 +60,13 @@ func (e *CatalogsVendor) Init(config vendors.VendorConfig, factories []managers.
 			var catalog model.CatalogState
 			jData, _ = json.Marshal(job.Body)
 			err = json.Unmarshal(jData, &catalog)
+			origin := event.Metadata["origin"]
 			if err == nil {
-				name := fmt.Sprintf("%s-%s", catalog.Spec.SiteId, catalog.Spec.Name)
```

Comment:
Is it safe to remove the SiteId property from Spec now?

Reply:
Yes. I have another commit to remove CatalogSpec.SiteId on my local branch. It depends on this PR and has a lower priority. I can create another PR once this one is merged.